source/tutorials/tutorials-dashboard-apps/dashboard.rst

```diff
@@ -0,0 +1,142 @@
+.. _app-development-tutorials-dashboard-apps-dashboard:
+
+Developing the Dashboard App
+============================
+
+.. warning::
+    You muse have followed :ref:`enabling-development-mode` for any part of this tutorial to work.
+
+Strategy
+--------
```

Comment:
Is `Strategy` meant to convey something different than the actual steps? Or is it like a summation before you get started? I'm a little confused on the purpose of this section.

That said - I did think a `Things to know` section may help (maybe that's what I thought this section was trying to convey?). Like general stuff to be aware of that doesn't necessarily fit anywhere else.

Reply:
Because the strategy applied to the shell as well, so it seemed like a pttern to call out overall is all. But, I'd be fine removing it. I don't like Things To Know sections myself, I prefer steps like this which layout the strategy, but maybe others would?

Reply:
I added a "Things to know" section. Let me know if that is what you had in mind, and feel free to enrich it. I tried to just call out anything that bit me starting out.

Reply:
I guess I'm just wondering how it's different from _the actual steps_ - is it an overview or something? 

And even then the notes are confusing when looking at the steps. Take symlinking for example it says:

https://github.com/OSC/ood-documentation/blob/b3a6a71f583ef2a8ad89f12e6d166f33912d8824/source/tutorials/tutorials-dashboard-apps/dashboard.rst?plain=1#L13

Then the steps say:
https://github.com/OSC/ood-documentation/blob/b3a6a71f583ef2a8ad89f12e6d166f33912d8824/source/tutorials/tutorials-dashboard-apps/dashboard.rst?plain=1#L23

The name of the section aside - it seems to me that the document could use some complimentary information around _why you're doing any given step_.  Like we're symlinking because  `OOD will try to recognize any directory in ~/ondemand/dev as an application`.